crud/common/schema.lua

```diff
@@ -244,4 +247,20 @@ function schema.result_needs_reload(space, result)
     return result.space_schema_hash ~= get_space_schema_hash(space)
 end
 
+function schema.fetch_ddl_sharding_key(box_obj)
+    dev_checks('table')
+
+    local sharding_key_space = box_obj.space['_ddl_sharding_key']
+    local sharding_key = {}
+    if sharding_key_space ~= nil then
+        for _, tuple in pairs(sharding_key_space:select()) do
+            local space_name = tuple[1]
+            local key = tuple[2]
```

Comment:
Could you use field names from format here?

Reply:
Sure.
```lua
--- a/crud/common/schema.lua
+++ b/crud/common/schema.lua
@@ -254,8 +254,8 @@ function schema.fetch_ddl_sharding_key(conn)
     local sharding_key = {}
     if sharding_key_space ~= nil then
         for _, tuple in pairs(sharding_key_space:select()) do
-            local space_name = tuple[1]
-            local key = tuple[2]
+            local space_name = tuple['space_name']
+            local key = tuple['sharding_key']
             sharding_key[space_name] = key
         end
     end
```

Reply:
To avoid misunderstanding I have used variables with "speaking" names: `space_name` and `key`.